tests/test_data.py

```diff
@@ -0,0 +1,97 @@
+import random
+import string
+import time
+
+import pytest
+
+from pgactivity.data import Data
+
+
+@pytest.fixture
+def data(postgresql):
+    return Data.pg_connect(
+        1,
+        host=postgresql.info.host,
+        port=postgresql.info.port,
+        database=postgresql.info.dbname,
+        user=postgresql.info.user,
+    )
+
+
+@pytest.fixture
+def tmp_table(postgresql):
+    tablename = "".join(random.choice(string.ascii_lowercase) for _ in range(4))
+    with postgresql.cursor() as cur:
+        cur.execute(f"CREATE TABLE {tablename} (s text)")
+    postgresql.commit()
+    return tablename
+
+
+def test_pg_is_local(postgresql, data):
+    assert data.pg_is_local()
+
+
+def test_pg_is_local_access(postgresql, data):
+    assert data.pg_is_local_access()
+
+
+def test_pg_get_db_info(data):
+    dbinfo = data.pg_get_db_info(None)
+    assert set(dbinfo) == {
+        "timestamp",
+        "no_xact",
+        "total_size",
+        "max_length",
+        "tps",
+        "size_ev",
+    }
+
+
+def test_activities(postgresql, data, execute):
+    with postgresql.cursor() as cur:
+        cur.execute("SELECT pg_sleep(1)")
+    (running,) = data.pg_get_activities()
+    assert "pg_sleep" in running.query
+    assert running.state == "idle in transaction"
+    assert running.wait == "ClientRead"
```

Comment:
We need a version test here since the output is different before pg 10: 

```suggestion
    if postgresql.server_version >= 100000:
        assert running.wait == "ClientRead"
    else:                                                                       
        assert running.wait == None
```

Reply:
This should not have slipped in since we're supposed to run tests with postgres version 9.6 to 13. But there's a configuration mistake, so this wasn't true. Adding a fix in the PR, thanks.

Reply:
Hm, in fact, it's totally broken and fixing this is beyond the scope of the PR so I'll disable the invalid CI setup here until I find time to improve it.